rpc/test/helpers.go

```diff
@@ -79,7 +79,9 @@ func makePathname() string {
 	}
 	// fmt.Println(p)
 	sep := string(filepath.Separator)
-	return strings.ReplaceAll(p, sep, "_")
+	// Add timestamp to make it unique for each test run to avoid reusing temp directories
+	baseName := strings.ReplaceAll(p, sep, "_")
+	return fmt.Sprintf("%s_%d", baseName, time.Now().UnixNano())
```

Comment:
revert this because this is used as part of a temporary directory which should be unique:
```
rootDir, err := os.MkdirTemp("", fmt.Sprintf("%s-%s_", chainID, testName))
```